example/spec/parallel/ingestGranule/IngestGranuleSuccessSpec.js

```diff
@@ -484,35 +495,42 @@ describe('The S3 Ingest Granules workflow', () => {
           workflow: 'PublishGranule'
         });
 
+        const publishGranuleExecution = await waitForTestExecutionStart(
+          'PublishGranule',
+          config.stackName,
+          config.bucket,
+          findExecutionForGranule,
+          { granuleId: inputPayload.granules[0].granuleId }
+        );
+
+        console.log(`Wait for completed execution ${publishGranuleExecution.executionArn}`);
```

Comment:
This doesn't look like a debugging statement, but I wanted to check.

Reply:
Yeah it's nice to have the ARNs in the console output so it's easily traceable if something goes wrong. Thank you for checking!